src/internal/packager/sbom/catalog.go

```diff
@@ -71,6 +72,11 @@ func Catalog(componentSBOMs map[string]*layout.ComponentSBOM, imageList []transf
 	for _, refInfo := range imageList {
 		builder.spinner.Updatef("Creating image SBOMs (%d of %d): %s", currImage, imageCount, refInfo.Reference)
 
+		if strings.HasSuffix(refInfo.Tag, ".sig") || strings.HasSuffix(refInfo.Tag, ".att") {
```

Comment:
You could try again looking at the `img` object's MediaType that is returned from `utils.LoadOCIImage` to check if we should skip or not.

Reply:
We may also want to say "non-image artifact" as well since that check wouldn't be cosign specific anymore